splink/comparison_level_creator.py

```diff
@@ -0,0 +1,98 @@
+from abc import ABC, abstractmethod
+from typing import final
+
+from .comparison_level import ComparisonLevel
+from .input_column import InputColumn
+
+
+class ComparisonLevelCreator(ABC):
+    def __init__(self, col_name: str = None):
+        """_summary_
+
+        Args:
+            col_name (str): _description_
+            dialect (str, optional): _description_. Defaults to None.
+            m_probability (_type_, optional): _description_. Defaults to None.
+        """
+        self.col_name = col_name
+        # TODO: do we need to set these to None, or let ComparisonLevel handle missings?
+        self.m_probability = None
```

Comment:
Yeah i think you're right - these are probably not needed.  They are a relic from when i was wondering about whether hte user should also be able to do like `LevenshteinLevel.m_probability = 0.9`, but I don't think we're envisaging users should do that




They're probably unnecessary.